cloudinit/sources/DataSourceVMware.py

```diff
@@ -292,6 +344,543 @@ def redact_keys(self):
         if self.data_access_method == DATA_ACCESS_METHOD_GUESTINFO:
             guestinfo_redact_keys(keys_to_redact, self.vmware_rpctool)
 
+    def get_envvar_data_fn(self):
+        """
+        check to see if there is data via env vars
+        """
+        md, ud, vd = None, None, None
+        if os.environ.get(VMX_GUESTINFO, ""):
+            md_in_json_or_yaml = guestinfo_envvar("metadata")
+            ud = guestinfo_envvar("userdata")
+            vd = guestinfo_envvar("vendordata")
+
+            if md_in_json_or_yaml:
+                md = load_json_or_yaml(md_in_json_or_yaml)
+        return (md, ud, vd)
+
+    def get_guestinfo_data_fn(self):
```

Comment:
See previous comment about the `_fn` suffix.

Reply:
ditto.